docs/configuration/config-file-reference.md

```diff
@@ -5364,6 +5373,36 @@ time_window:
   # lookback delta) that the query should be within. If set to 0, it won't be
   # checked.
   [end: <int> | default = 0]
+
+# Limit that query time range should be within. If not set, it won't be checked.
+time_range_limit:
+  # Query time range should be above or equal to this value to match. If set to
+  # 0, it won't be checked.
+  [min: <int> | default = 0]
+
+  # Query time range should be below or equal to this value to match. If set to
+  # 0, it won't be checked.
+  [max: <int> | default = 0]
+
+# Limit that query step should be within. It will check subquery steps as well.
+# If not set, it won't be checked.
+query_step_limit:
+  # Query step should be above or equal to this value to match. If set to 0, it
+  # won't be checked.
+  [min: <int> | default = 0]
+
+  # Query step should be below or equal to this value to match. If set to 0, it
+  # won't be checked.
+  [max: <int> | default = 0]
+
+# User agent for the query. If not set, it won't be checked.
```

Comment:
Do we need to specify regex for user agent or it needs exact match?

Reply:
it needs exact match

Reply:
changed to regex